src/MongoDB.Bson/BsonConstants.cs

```diff
@@ -45,6 +45,14 @@ public static long DateTimeMaxValueMillisecondsSinceEpoch
             get { return __dateTimeMaxValueMillisecondsSinceEpoch; }
         }
 
+        /// <summary>
+        /// Gets the number of milliseconds since the Unix epoch for DateTime.MaxValue.
+        /// </summary>
+        public static long DateTimeMillisecondsStartEpoch
+        {
+            get { return 0; }
+        }
+
```

Comment:
I would not add this constant.

In any case the name does not seem logical.

Reply:
Done.